storage/Filesystems/MountPointImpl.cc

```diff
@@ -386,7 +386,7 @@ namespace storage
 	    }
 	}
 
-	if ((!lhs.active && active) || (lhs.path != path))
+	if (active && (!lhs.active || lhs.path != path))
```

Comment:
well, as it is obviously due to previous bug not trivial condition, I would like to see comment here explaining when exactly it mount/unmount.

Reply:
In fact, I think I have found a bug while documenting it :(